lib/services/files-api-service.js

```diff
@@ -0,0 +1,107 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+var di = require('di');
+var fs = require('fs');
+var path = require('path');
+var express = require('express');
+var app = express();
+var walking = require('walk');
+
+module.exports = filesApiServiceFactory;
+
+di.annotate(filesApiServiceFactory, new di.Provide('Http.Services.Api.Files'));
+di.annotate(filesApiServiceFactory,
+    new di.Inject(
+      'Promise',
+      'Services.Waterline'
+    )
+);
+
+
+function filesApiServiceFactory(
+  Promise,
+  waterline
+) {
+  var walker;
+  var walk = Promise.promisifyAll(walking);
+  var onhttpPath = __dirname.split('/');
+  onhttpPath.splice(0,2);
+  onhttpPath = onhttpPath.join("/");
+  Promise.promisifyAll(fs);
+
+  function FilesApiService() {
+  }
+
+  /**
+   * Get list of static files
+   * @returns {array}
+   */
+
+  FilesApiService.prototype.getAllStaticFiles = function() {
+
+    app.use(express.static(onhttpPath));
```

Comment:
That line is used to enable the relative paths I set for within on-http and needs to be updated. I found that the paths to static files were inconsistent with what I was told. For example, on my monorail server the default value of httpStaticRoot is `/opt/monorail/static/http` which does not exist in my instance. The paths below are the presumed likely places to check for static files.
The above line sets the working directory to be wherever the api call was sent from. In this case it will always be on-http